contracts/external/cw-fund-distributor/src/state.rs

```diff
@@ -0,0 +1,22 @@
+use cosmwasm_std::{Addr, Uint128};
+use cw_storage_plus::{Item, Map};
+use cw_utils::Expiration;
+
+// block height for distribution snapshot
```

Comment:
For auto generated docs it's best to use `///` here so that the comments show up in generated docs.